src/main/resources/uk/gov/ida/rp/testrp/views/layout/layout.jade

```diff
@@ -10,8 +10,6 @@ html(lang="en")
   head
     meta(http-equiv="content-type", content="text/html; charset=UTF-8")
     title=pageTitle
-    script.
-      (function(){if(navigator.userAgent.match(/IEMobile\/10\.0/)){var d=document,c="appendChild",a=d.createElement("style");a[c](d.createTextNode("@-ms-viewport{width:auto!important}"));d.getElementsByTagName("head")[0][c](a);}})();
```

Comment:
What happened to this?

Reply:
IEMobile 10 is not supported, at least it doesn't have full support

Reply:
Do we have a support list anywhere?

Reply:
https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices